LEGO1/mxbitmap.h

```diff
@@ -1,13 +1,24 @@
 #ifndef MXBITMAP_H
 #define MXBITMAP_H
 
+#include "legoinc.h"
+#include "mxpalette.h"
+
 class MxBitmap
 {
 public:
   __declspec(dllexport) MxBitmap();
   __declspec(dllexport) virtual ~MxBitmap();
   __declspec(dllexport) virtual MxPalette *CreatePalette();
   __declspec(dllexport) virtual long Read(const char *);
+private:
+  virtual void vtable();
```

Comment:
This is not how it works. Presence of a virtual function, in this class or any of its baseclasses, enforces the generation of a hidden vfptr at object offset 0 (disregarding multiple- and virtual inheritance). That pointer is generated in the top-most class having a virtual function. Since inheriting a type includes all its data, by inheriting from MxCore the vfptr becomes automatically included in MxBitmap.